conn.go

```diff
@@ -729,3 +727,20 @@ func (c *Conn) SendKeyUpdate(requestUpdate bool) error {
 func (c *Conn) GetHsState() string {
 	return reflect.TypeOf(c.hState).Name()
 }
+
+func (c *Conn) ComputeExporter(label string, context []byte, keyLength int) ([]byte, error) {
+	_, ok := c.hState.(StateConnected)
+	if !ok {
```

Comment:
That said, if you want to factor out the early exporter, then checking that the appropriate secret is non-nil will achieve the same goal.